CHANGELOG.md

```diff
@@ -30,6 +31,115 @@
+    architecture specific namely `.arm64.` and `.amd64.`
+
+### Additions
+
+##### Core
+
+  - :fireworks: New configuration option `cassandra_refresh_frequency` to set
+    the frequency that Kong will check for Cassandra cluster topology changes,
+    avoiding restarts when Cassandra nodes are added or removed.
+    [#5071](https://github.com/Kong/kong/pull/5071)
+  - New `transformations` property in DAO schemas, which allows adding functions
+    that run when database rows are inserted or updated.
+    [#5047](https://github.com/Kong/kong/pull/5047)
+  - The new attribute `hostname` has been added to `upstreams` entities. This
+    attribute is used as the `Host` header when proxying requests through Kong
+    to servers that are listening on server names that are different from the
+    names to which they resolve.
+    [#4959](https://github.com/Kong/kong/pull/4959)
+  - New status interface has been introduced. It exposes insensitive health,
+    metrics and error read-only information from Kong, which can be consumed by
+    other services in the infrastructure to monitor Kong's health.
+    This removes the requirement of the long-used workaround to monitor Kong's
+    health by injecting a custom server block.
+    [#4977](https://github.com/Kong/kong/pull/4977)
+  - New Admin API response header `X-Kong-Admin-Latency`, reporting the time
+    taken by Kong to process an Admin API request.
+    [#4966](https://github.com/Kong/kong/pull/4996/files)
+
+##### Configuration
+
+  - :warning: New configuration option `service_mesh` which enables or disables
+    the Service Mesh functionality. The Service Mesh is being deprecated and
+    will not be available in the next releases of Kong.
+    [#5124](https://github.com/Kong/kong/pull/5124)
+  - New configuration option `router_update_frequency` that allows setting the
+    frequency that router and plugins will be checked for changes. This new
+    option avoids performance degradation when Kong routes or plugins are
+    frequently changed. [#4897](https://github.com/Kong/kong/pull/4897)
+
+##### Plugins
+
+  - rate-limiting: in addition to consumer, credential, and IP levels, now
+    rate-limiting plugin has service-level support. Thanks
+    [wuguangkuo](https://github.com/wuguangkuo) for the patch!
+    [#5031](https://github.com/Kong/kong/pull/5031)
+  - Now rate-limiting `local` policy counters expire using the shared
+    dictionary's TTL, avoiding to keep unnecessary counters in memory. Thanks
+    [cb372](https://github.com/cb372) for the patch!
+    [#5029](https://github.com/Kong/kong/pull/5029)
+  - Authentication plugins have support for tags now.
+    [#4945](https://github.com/Kong/kong/pull/4945)
+  - response-transformer plugin now supports renaming response headers. Thanks
+    [aalmazanarbs](https://github.com/aalmazanarbs) for the patch!
+    [#5040](https://github.com/Kong/kong/pull/5040)
+  - aws-lambda plugin now automatically loads credentials
```

Comment:
This entry doesn't have an associated PR link — is this really in 1.4.0rc2? I couldn't find it. Perhaps this is related to the aws-lambda bump which is in `next` (but isn't in 1.4.0rc2 because it was added after the 1.4.0 feature freeze?)

Reply:
Yes, you're right. I was tricked by it being merged in `next`.